deploy/install_full.bash

```diff
@@ -54,22 +54,24 @@ verify_dependencies() {
 }
 
 
-install_snapd_on_centos7() {
+install_dependencies_on_centos7() {
   yum -y install epel-release
   yum -y install snapd
   systemctl enable snapd
   systemctl start snapd
   sleep 10
   ln -s /var/lib/snapd/snap /snap
+  yum -y install bind-utils
 }
 
-install_snapd_on_centos8() {
+install_dependencies_on_centos8() {
   dnf -y install epel-release
   dnf -y install snapd
   systemctl enable snapd
   systemctl start snapd
   sleep 10
   ln -s /var/lib/snapd/snap /snap
+  yum -y install bind-utils
```

Comment:
see above

Reply:
No I dont need as nslookup is command not service as snmpd